.changeset/rich-horses-begin.md

```diff
@@ -0,0 +1,26 @@
+---
+'astro': major
+---
+
+Removes the deprecated `<ViewTransitions />` component
+
+In Astro 5.0, the `<ViewTransitions />` component was renamed to `<ClientRouter />` to clarify the role of the component. The new name makes it more clear that the features you get from Astro's `<ClientRouter />` routing component are slightly different from the native CSS-based MPA router. However, a deprecated version of the `<ViewTransitions />` component still existed and may have functioned in Astro 5.x.
+	
+Astro 6.0 removes the `<ViewTransitions />` component entirely and can no longer be used in your project. Update to the `<ClientRouter />` component to continue to use these features.
```

Comment:
```suggestion
Astro 6.0 removes the `<ViewTransitions />` component entirely and it can no longer be used in your project. Update to the `<ClientRouter />` component to continue to use these features.
```